gslib/tests/test_acl.py

```diff
@@ -272,155 +273,179 @@ def test_acl_set_version_specific_uri(self):
   def _strip_json_whitespace(self, json_text):
     return re.sub(r'\s*', '', json_text)
 
+  def _MakeScopeRegex(self, role, entity_type, email_address):
+    template_regex = (r'\{.*"entity":\s*"%s-%s".*"role":\s*"%s".*\}' %
+                      (entity_type, email_address, role))
+    return re.compile(template_regex, flags=re.DOTALL)
+
+  def _MakeProjectScopeRegex(self, role, project_team, project_number):
+    template_regex = (
+        r'\{.*"entity":\s*"project-%s-%s",\s*"projectTeam":\s*\{\s*"'
+        r'projectNumber":\s*"%s",\s*"team":\s*"%s"\s*\},\s*"role":\s*"%s".*\}' %
+        (project_team, project_number, project_number, project_team, role))
+
+    return re.compile(template_regex, flags=re.DOTALL)
+
   def testAclChangeWithUserId(self):
```

Comment:
Why was this change necessary?

Reply:
Previously these tests were basically just unit tests of the translation functionality. Now they run end to end so we can use them to confirm parity. Was there a more specific change you were referring to?

Reply:
Got it